circle.yml

```diff
@@ -123,6 +123,7 @@ jobs:
           command: |
             apt-get -qq update
             apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libz3-dev
+            ./scripts/install_jsoncpp.sh
```

Comment:
Please name this something more accurate, such as `install_obsolete_jsoncpp_1_7_4.sh`.